proxy-libs/README.md

```diff
@@ -0,0 +1,12 @@
+# proxy-libs
+
+These are the supported libraries for auto instrumentation in go.
+
+Release supported tag for proxy-libs
+-
+``` git tag github.com/helios/go-sdk/proxy-libs/<PROXY_LIB_NAME>/<TAG> ```
+
+For example:
+``` git tag github.com/helios/go-sdk/proxy-libs/heliosgrpc/v1.51.0```
+Then, push the new tag:
+``` git push ```
```

Comment:
`git push` will push all the local tags and branches
use: `git push origin github.com/helios/go-sdk/proxy-libs/heliosgrpc/v1.51.0` instead